cranelift/object/src/backend.rs

```diff
@@ -551,6 +551,24 @@ impl ObjectModule {
                     symbol
                 }
             }
+            ir::ExternalName::TlsIndex => {
+                let name = b"_tls_index";
+                self.object.symbol_id(name).unwrap_or_else(|| {
+                    self.object.add_symbol(Symbol {
+                        name: name.to_vec(),
+                        value: 0,
+                        size: 32,
+                        // TODO: Is this correct?
+                        kind: SymbolKind::Tls,
```

Comment:
I did expect it to not be.

Reply:
Looks like object doesn't distinguish between text, data and tls for COFF: https://github.com/gimli-rs/object/blob/404ae26dac1eafc82ecad4f02ba8ccda57f7cb27/src/write/coff.rs#L597 And I couldn't find anywhere COFF distinguishes between data and tls undefined symbols.